package.json

```diff
@@ -173,6 +174,7 @@
     "pidtree": "^0.5.0",
     "prettier": "^1.18.2",
     "seedrandom": "^3.0.5",
+    "sinon": "^9.0.3",
```

Comment:
Sinon `9` dropped support for Node 8: https://github.com/sinonjs/sinon/blob/master/CHANGELOG.md#900--2020-02-19

Reply:
🤦  Good catch. I guess tests are passing on node 8 since I didn't use any API that requires a newer version.
I'll update it

Reply:
Fixed it as a part of the rebase https://github.com/netlify/cli/pull/1215/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R176